server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -229,6 +229,13 @@ write_read_schema_free_memory_proportion=4:3:1:2
 # primitive array size (length of each array) in array pool
 primitive_array_size=128
 
+
+# time indexer
+enable_file_time_indexer=false
```

Comment:
Currently this parameter is false. We should open it and test it's functionality and performance